### PR TITLE
Qa 577 add CI section

### PR DESCRIPTION
**Pull request by galeza**

### CODE REVIEW

File: src/data/roadmapData.json [hunk 1]
```diff
@@ -422,12 +422,74 @@
...
       "CI Integration": [
         {
-          "name": "placeholder",
+          "name": "Basic CI knowledge",
           "data": [
             {
-              "description": "description 1",
-              "url": "http://google.com",
-              "urlLabel": "some urlLabel"
+              "description": "Describes Continues Integration. The artcle contains several links to the useful resources.",
```
Comment (woytekn) [inline src/data/roadmapData.json:428]: "Continues" - typo

File: src/data/roadmapData.json [hunk 2]
```diff
@@ -422,12 +422,74 @@
...
+              "url": "https://www.atlassian.com/continuous-delivery/tutorials/continuous-integration-tutorial",
+              "urlLabel": "Basic knowledge about Continues Integration"
```
Comment (woytekn) [inline src/data/roadmapData.json:430]: "Continues" - typo

File: src/data/roadmapData.json [hunk 3]
```diff
@@ -422,12 +422,74 @@
...
+            },
+            {
+              "description": "Presents types of tests and CI. Describes how to adopt Continues Integration",
```
Comment (woytekn) [inline src/data/roadmapData.json:433]: "Continues" - typo

**Review by woytekn — CHANGES_REQUESTED** [review]

File: src/data/roadmapData.json (hunk 1 above)
Comment (galeza) [reply src/data/roadmapData.json]: Ok, fixed

File: src/data/roadmapData.json (hunk 2 above)
Comment (galeza) [reply src/data/roadmapData.json]: Ok, fixed

File: src/data/roadmapData.json (hunk 3 above)
Comment (galeza) [reply src/data/roadmapData.json]: Ok, fixed

File: src/data/roadmapData.json [hunk 4]
```diff
@@ -422,12 +422,74 @@
...
+        {
+          "name": "Continuous Integration with Jenkins",
+          "data": [
+            {
+              "description": "This article describes how to use the Jenkins continuous integration build server.",
+              "url": "https://www.vogella.com/tutorials/Jenkins/article.html",
+              "urlLabel": "Continuous integration with Jenkins - Tutorial"
+            },
+            {
+              "description": "Continuous Integration with Jenkins",
```
Comment (zciluk) [inline src/data/roadmapData.json:464]: I would add some more description here to distinct it more from the first link provided. Like "It's a video course that teaches A B and C"

File: src/data/roadmapData.json [hunk 5]
```diff
@@ -422,12 +422,74 @@
...
+              "url": "https://testautomationu.applitools.com/jenkins-tutorial/",
+              "urlLabel": "Test Automation University"
+            }            
+          ]
+        },
+        {
+          "name": "Run Cypress on CI - generic CI, GitHub Actions, CircleCI, Netlify",
+          "data": [
+            {
+              "description": "Learn how to run Cypress on a generic CI, on GitHub Actions, on CircleCI, and on Netlify. You need a free acount.",
```
Comment (zciluk) [inline src/data/roadmapData.json:474]: It would be nice to include here Parts in description - Part 1 and Part 2 to clearly distinct them

**Review by zciluk — CHANGES_REQUESTED** [review]
Very nice links, only some minor changes requested.

**Review by zciluk — APPROVED** [review]
LGTM

**Review by woytekn — APPROVED** [review]

**Pull request merged**